.rubocop.yml

```diff
@@ -13,55 +13,10 @@ AllCops:
     - vendor/**
     - package/**
 
-ClassLength:
-  Max: 128
-
-HashSyntax:
-  Enabled: false
-
-MethodLength:
-  Max: 15
-
-LineLength:
-  Max: 160
-
-SpaceBeforeBlockBraces:
-  Enabled: true
-  EnforcedStyle: space
-
-SpaceInsideBlockBraces:
-  EnforcedStyle: space
-  SpaceBeforeBlockParameters: false
-
-EmptyLinesAroundMethodBody:
-  Enabled: false
-
-EmptyLinesAroundClassBody:
-  Enabled: false
-
-EmptyLinesAroundModuleBody:
-  Enabled: false
-
-SignalException:
-  Enabled: false
-
-DoubleNegation:
-  Enabled: false
-
-PercentLiteralDelimiters:
-  PreferredDelimiters:
-    '%w': '{}'
-
-SpaceBeforeFirstArg:
+Rails:
```

Comment:
@thutterer 

> Plus, disables all Rails cops, as this is not a Rails application

That's why in Rubocop by default the Rails cop is disabled! You can remove the `Rails` cop from this fail, it doesn't have any effect. 😉

Reply:
@Ana06 We have enabled the Rails cops in the [gem](https://github.com/SUSE/scc-codestyle/blob/master/default.yml#L426) as most of our apps are Rails. But yeah, it might be cleaner to flip that around, so that the gem does not force the Rails-specific stuff to be turned on, but let the individual app decide if it cares about Rails cops or not.

Thanks for the comment ... how have you found this PR anyway? :female_detective:

Reply:
@eduardoj sent it to me... it seems he is following this project as he finds it interesting. 😄  and he knows I would like to see that people is using Rubocop default configuration 😝